statsmodels/discrete/tests/test_sandwich_cov.py

```diff
@@ -479,13 +479,9 @@ def test_basic(self):
         assert_equal(res1.cov_type, self.cov_type)
         assert_equal(res2.cov_type, self.cov_type)
 
-        assert_allclose(res1.params, res2.params, rtol=1e-13)
-        # bug TODO res1.scale missing ?  in Gaussian/OLS
-        assert_allclose(res1.bse, res2.bse, rtol=1e-13)
-#         if not self.cov_type == 'nonrobust':
-#             assert_allclose(res1.bse * res1.scale, res2.bse, rtol=1e-13)
-#         else:
-#             assert_allclose(res1.bse, res2.bse, rtol=1e-13)
+        rtol = getattr(res1, 'rtol', 1e-13)
```

Comment:
should this be getting `cls.rtol`?  That is the attribute that is set down on L514.

Reply:
Yes, that's a mistake.
However, why do the tests not fail if rtol is 1e-13?

Essentially this means that GLM and the corresponding discrete model agree on very high precision with the same optimizer newton. 1e-13 should be much stricter than convergence tolerance.